requirements.txt

```diff
@@ -1,2 +1,3 @@
 pypif==2.1.0
 xlrd
+pytest
```

Comment:
This would typically go in a `test_requirements.txt` file, since its not an install requirement.

Reply:
I can do that